fastparquet/api.py

```diff
@@ -993,7 +993,7 @@ def _dtypes(self, categories=None):
                         if self.pandas_nulls:
                             dtype[col] = converted_types.nullable[dt]
                         else:
-                            dtype[col] = np.float_()
+                            dtype[col] = np.float64_()
```

Comment:
Isn't it `np.float64()` (no underscore suffix) ?

Reply:
woups, typo, nice catch ! thx